### ISSUE TITLE
Object with encrypted field does not load

### ISSUE THREAD

**Issue by iibs** [DivanteLtd/pimcore-translation-bundle#21]
Hi,

If I have this bundle enabled and an object class has an encrypted input field in it, I cannot open it.

Console gives following error:
```
TypeError: Cannot read property 'language' of undefined
```

Problem seems to be in input.js, row 51:

```
if (this.context.language) {
  ...
}
```

Changing the if to `if(this.context && this.context.language)` seems to fix the issue.

**Comment by iibs** [comment]
@kubaplas I created pull request! I added the context check also for textarea and wysiwyg, since I assume the problem can occur in those too.

**Comment by kubaplas** [comment]
Thank you @iibs ! Pull request merged and a new release 1.2.6 created!
